lib/workos/directory_sync/directory_sync.ex

```diff
@@ -1,5 +1,5 @@
 defmodule WorkOS.DirectorySync do
-  alias WorkOS.Api
+  import WorkOS.API
```

Comment:
Makes sense to use `import` here in order to access functions from the API module without using the fully-qualified name

I do think that accessing `API.get` isn't bad for readability but still not that necessary